packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -36,21 +41,42 @@ export interface DateInputProps extends Omit<InputProps, OmittedInputProps> {
    */
   description?: React.ReactNode
   isOpen: boolean
+  /**
+   * Event for the onClick of the icon button
+   */
   onButtonClick: () => void
   isReversed?: boolean
+  /**
+   * The input value as a Date
+   */
   valueDate: Date | undefined
-  onBlur: (date: Date | undefined) => void
+  /**
+   * The callback for then onBlur is triggered on the input
+   */
+  onBlur: (date: Date | undefined, inputValue: string) => void
   disabledDays?: Modifier | Modifier[]
+  /**
+   * A descriptive message for `status` states
+   */
+  validationMessage?: string | React.ReactNode
+  /**
+   * Updates the styling of the validation FieldMessage.
```

Comment:
```suggestion
   * Updates the styling of the validation FieldMessage
```

I don't remember if we have `.` or not, but best to be consistent.